Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -860,6 +860,16 @@ extension HTTPClient.Configuration {
         /// - warning: Cycle detection will keep all visited URLs in memory which means a malicious server could use this as a denial-of-service vector.
         public static func follow(max: Int, allowCycles: Bool) -> RedirectConfiguration { return .init(configuration: .follow(max: max, allowCycles: allowCycles)) }
     }
+
+    /// Connection pool configuration.
+    public struct PoolConfiguration: Hashable {
+        // Specifies amount of time connections are kept idle in the pool.
+        var idleTimeout: TimeAmount
```

Comment:
Whoops, nearly missed this.

```suggestion
        public var idleTimeout: TimeAmount
```

Reply:
🙏

Reply:
fixed!